src/core/src/sim/mc/monte_carlo.rs

```diff
@@ -194,6 +197,17 @@ impl Propagator for MonteCarlo {
             mcmove.1.nattempted = 0;
         }
     }
+
+    /// Print some informations about moves to screen
+    fn finish(&mut self, _: &System) {
+        println!("Monte Carlo simulation summary");
```

Comment:
You should use `info!` here, as the standard output may not exist, or the user may want to redirect everything to a file.